### ISSUE TITLE
Assumes Coalesce Communication Structure

### ISSUE THREAD

**Issue by amark** [amark/theory#1]
All code is essentially three things:
1. Data, 2. Manipulation, 3. Communication.

As a consequence, after prolonged thought, wrestling back and forth, and switching sides numerously, I, the creator of this library, have come to the definitive conclusion that the communication component is, and ought, to remain core.

Choosing to do this also means I have to maintain this decision as well, so it does not come without consequences for me.

Theory is tiny, if you do not need this module, keep it but do not use it.
Unofficially, if byte size is life or death vital, then you can erase the entirety of the component under two conditions:
1. You are not allowed to let others use your modified copy of Theory, only you can, as all official copies of Theory must include the full core components, including communication.
2. You accept that any code that depends upon it will break, probably causing everything to break.

Because communication requires an API, Theory also assumes the messaging structure as proposed by the Coalesce framework. This is what brought me the most grief about whether it should be considered core or not. But because I feel strongly that the future of code will be more entangled with communication than not, and that this structure is the most open, I have permanently decided to depend upon it as default.

Unofficially, with the same disclaimers and conditions previously stipulated, you can overwrite, in many different ways, the communication, whether through the configuration options or not using the helper functions.

In conclusion, the communication component is core, because communication is essential for code. Thus sayeth the author, and it was good.

Amen.

**Comment by amark** [comment]
The above still stands.
However, if https://github.com/amark/theory/issues/3 is implemented, there should be an easy and handy way to build a custom version of Theory without the communication module. But these custom builds must only be used by you.

Also, you can always disable communication with `theory.com.off = true;` in the theory script tag, where you do configurations.
